### ISSUE TITLE
jq: error: Could not open file /data/options.json: Permission denied

### ISSUE THREAD

**Issue by mtthp** [matt-FFFFFF/hassio-addon-repository#8]
After updating the HA supervisor, it seems that the `options.json` file is out of reached because the teslamate addon isn't run with the root user, as it is stated here : https://github.com/home-assistant/supervisor/issues/2158

Here are the logs of my instance : 
```
jq: error: Could not open file /data/options.json: Permission denied
jq: error: Could not open file /data/options.json: Permission denied
jq: error: Could not open file /data/options.json: Permission denied
jq: error: Could not open file /data/options.json: Permission denied
jq: error: Could not open file /data/options.json: Permission denied
jq: error: Could not open file /data/options.json: Permission denied
jq: error: Could not open file /data/options.json: Permission denied
jq: error: Could not open file /data/options.json: Permission denied
jq: error: Could not open file /data/options.json: Permission denied
jq: error: Could not open file /data/options.json: Permission denied
jq: error: Could not open file /data/options.json: Permission denied
jq: error: Could not open file /data/options.json: Permission denied
jq: error: Could not open file /data/options.json: Permission denied
jq: error: Could not open file /data/options.json: Permission denied
jq: error: Could not open file /data/options.json: Permission denied
/ha-bootstrap.sh: line 24: [: ==: unary operator expected
waiting for postgres at 127.0.0.1:5432 (2sec)
```

Is it possible to set the user as root in https://github.com/matt-FFFFFF/hassio-addon-repository/blob/master/teslamate/Dockerfile#L40 to fix the issue ?

**Comment by matt-FFFFFF** [comment]
Understood, leave it with me - thanks for reporting

**Comment by matt-FFFFFF** [comment]
Hi, have released 0.4.0 - please can you validate @mtthp ?

**Comment by mtthp** [comment]
Yes, working as it should now. I tested by pulling locally your repo because it doesn't seems to be available by fetching it from Github.
Thank you very much for your help 👍
